include/libtorrent/alert_types.hpp

```diff
@@ -533,13 +528,23 @@ TORRENT_VERSION_NAMESPACE_2
 		static constexpr alert_category_t static_category = alert_category::tracker | alert_category::error;
 		std::string message() const override;
 
+		// This member says how many times in a row this tracker has failed.
 		int const times_in_row;
+
+		// the error code indicating why the tracker announce failed. If it is
+		// is ``lt::errors::tracker_failure`` the failure_reason() might contain
+		// a more detailed description of why the tracker rejected the request.
+		// HTTP status codes indicating errors are also set in this field.
```

Comment:
What about other kinds of errors? Like `Host not found (non-authoritative)`? They have to be checked elsewhere, right? That should probably be mentioned here.

Reply:
I don't really want to enumerate *all* categories of errors that can happen. But yes, all errors that can happen will be indicated by this error code

Reply:
Never mind, I misunderstood something, I also don't think all possible categories should be enumerated here.